packages/web/src/scss/components/FileUploader/_FileUploaderList.scss

```diff
@@ -1,5 +1,17 @@
+// 1. Avoid unwanted extra spacing by moving the space on the first child since the list is always present in the DOM,
+//    even when empty. (Surprisingly, we cannot use the `:empty` pseudo-selector here because it doesn't trigger when
+//    there are white-space characters inside the list. The white space can be avoided in HTML and Twig, but not with
+//    our JS plugin.)
```

Comment:
This seems to happen only in Firefox:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/ae0e0fd6-4803-4592-846e-a2da011e5a3b)

Can we fix it in the JS plugin?